src/main/java/novamachina/exnihilosequentia/common/compat/kubejs/SimpleChanceRecipeJS.java

```diff
@@ -0,0 +1,38 @@
+package novamachina.exnihilosequentia.common.compat.kubejs;
+
+import com.google.gson.JsonArray;
+import dev.latvian.mods.kubejs.item.ItemStackJS;
+import dev.latvian.mods.kubejs.util.ListJS;
+
+public class SimpleChanceRecipeJS extends BaseRecipeJS {
+
+  @Override
+  public void create(ListJS args) {
+    this.inputItems.add(this.parseIngredientItem(args.get(0)));
```

Comment:
i have to add something here too: we need to add those blocks to their equivalent BlockTag ("mineable/crook" and "mineable/hammer")